ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/fullyprivate/proxy/provision/cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy-provision-chain.yaml

```diff
@@ -11,6 +11,7 @@ chain:
   - chain: ipi-install
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
+  - chain: cucushift-installer-check-azure-fullyprivate
```

Comment:
Shall we add the check chain to other fullyprivate install workflow?

Reply:
Since it impacts many jobs including storage pre-submit jobs, I want to add one to see if it is stable, will add check chain to other workflow if it works for a while.